### PR TITLE
chore:Restructure file, install and configure Tailwind, create a simp…

### PR DESCRIPTION
**Pull request by LH-Song**
Commit: Restructure file organization, install and configure Tailwind CSS, and create a simple example page demonstrating how to use Tailwind CSS

### CODE REVIEW

**Comment by LH-Song** [comment]
@Qiming-Liu Alan老师，这里改了些eslint 主要原因是 prettier 和 formatting 不同意的问题，都是逗号要不要加 ，空格， 单双引引起的，暂时disable，等airbnb merge 就去搞统一

File: .stylelintrc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "extends": "stylelint-config-standard-scss",
+  "extends": "stylelint-config-tw",
```
Comment (Qiming-Liu) [inline .stylelintrc:2]: I didn't find a package called `stylelint-config-tw`.
I found [stylelint-config-tailwindcss](https://github.com/zhilidali/stylelint-config-tailwindcss), but it seems to have not much rules in it.
I advise that just use `stylelint-config-standard` here if you don't want to use sass.

File: src/app/TestPage/TestPage.tsx
Comment (Qiming-Liu) [inline src/app/TestPage/TestPage.tsx:1]: This file is actually in the router folder. I suggest renaming the app folder to router, and then put TestPage and HomePage under the Pages folder.

File: src/app/HomePage/HomePage.tsx
Comment (Qiming-Liu) [inline src/app/HomePage/HomePage.tsx:1]: Same to TestPage

File: .yarnrc.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (Qiming-Liu) [inline .yarnrc.yml:1]: This seems to be a default configuration. Do we need this file?

**Review by Qiming-Liu — CHANGES_REQUESTED** [review]

File: .stylelintrc (hunk 1 above)
Comment (LH-Song) [reply .stylelintrc:2]: updated to  "extends": "stylelint-config-standard-scss",

File: .yarnrc.yml (hunk 2 above)
Comment (LH-Song) [reply .yarnrc.yml:1]: deleted

File: src/app/TestPage/TestPage.tsx
Comment (LH-Song) [reply src/app/TestPage/TestPage.tsx:1]: updated

**Review by Qiming-Liu — APPROVED** [review]

**Pull request merged**